.circleci/config.yml

```diff
@@ -25,20 +25,18 @@ jobs:
               $UNDERLAY_WS/ros2_dependencies.repos
             vcs import src < ros2_dependencies.repos
             rosdep install -y --ignore-src --skip-keys "slam_toolbox" --from-paths src
-            colcon build --symlink-install --packages-up-to nav2_costmap_2d
-            source $UNDERLAY_WS/install/local_setup.bash
+            colcon build --symlink-install --packages-up-to nav2_costmap_2d rcpputils rosbag2_cpp
+            source $UNDERLAY_WS/install/setup.bash
             cd $OVERLAY_WS && rosdep install -y --ignore-src --from-paths src
       - run:
           name: Debug Build
           command: |
-            source `find $ROS_WS -maxdepth 2 -name local_setup.bash | sort | head -1`
-            source $UNDERLAY_WS/install/local_setup.bash
+            source $UNDERLAY_WS/install/setup.bash
```

Comment:
Is this line redundant? `underlay_ws` is already being sourced above. Same potential duplication with the sourcing in "Run Tests" below.